client/src/featureform/resources.py

```diff
@@ -453,12 +453,12 @@ def serialize(self) -> bytes:
 @typechecked
 @dataclass
 class MongoDBConfig:
-    username: str
-    password: str
-    host: str
-    port: str
-    database: str
-    throughput: int
+    username: str = ""
```

Comment:
do we really want users to be able to not pass username or password to any of these?

Reply:
and all the other fields

Reply:
on second thought good point. these should be required fields. I'll edit and see if anything blows up.